src/Magnum/DartIntegration/Test/ConvertShapeNodeTest.cpp

```diff
@@ -436,8 +436,18 @@ void ConvertShapeNodeTest::urdf() {
                 CORRADE_COMPARE(shapeDataAll->materials[0].diffuseColor(), (Vector3{0.6f, 0.6f, 0.6f}));
                 CORRADE_COMPARE(shapeDataMaterial->materials[0].diffuseColor(), (Vector3{0.6f, 0.6f, 0.6f}));
             } else {
+                #if DART_MAJOR_VERSION > 6 || ((DART_MAJOR_VERSION == 6) && (DART_MINOR_VERSION >=9))
+                if(shapeNode->getName() == "base_link_ShapeNode_0") {
+                    CORRADE_COMPARE(shapeDataAll->materials[0].diffuseColor(), 0x0_srgbf);
```

Comment:
`0x000000_srgbf` pretty please :) I know it's the same but I'd like to follow how CSS colors look. Same below.

Reply:
OK :)